guide/additional-info/changes-in-v13.md

```diff
@@ -0,0 +1,1095 @@
+
+### Webpack
+
+Webpack builds are no longer supported.
+
+## Changes and deletions
+
+### APIMessage
+
+The APIMessage class has been renamed to `MessagePayload`, resolving a naming clash with an interface in the `discord-api-types` library which raw message data objects.
+
+### Client
+
+#### Client#emojis
+
+The Client Emoji manager is now a `BaseGuildEmojiManager`, providing cache resolution only and removing methods that would fail to create emojis as there was no Guild context.
+
+#### Client#fetchApplication
+
+The `Client#fetchApplication` has been removed and replaced with the `Client#application` property.
+
+```diff
+- client.fetchApplication().then(application => console.log(application.name))
++ console.log(client.application.name);
+```
+
+#### Client#generateInvite
+
+`Client#generateInvite` no longer supports `PermissionsResolvable` as its argument, requiring `InviteGenerationOptions`.
+
+`InviteGenerationOptions` requires that at least one of either `bot` or `applications.commands` is provided in `scopes` to generate a valid invite URL.
+
+To generate an invite link for Slash Commands only.
+
+```js
+client.generateInvite({ scopes: ['applications.commands'] });
+```
+
+To generate an invite link for a bot and define required permissions.
+
+```diff
+- client.generateInvite([Permissions.FLAGS.SEND_MESSAGES]);
++ client.generateInvite({ scopes: ['bot'], permissions: [Permissions.FLAGS.SEND_MESSAGES] })
+```
+
+### Client#login
+
+Previously when a token had reached its 1000 login limit for the day, discord.js would treat this as a rate limit and silently wait to login again, but this was not communicated to the user.
+
+This will now instead cause an error to be thrown.
+
+### ClientOptions
+
+#### ClientOptions#fetchAllMembers
+
+The `ClientOptions#fetchAllMembers` option has been removed.
+
+With the introduction of gateway intents, the `fetchAllMembers` Client option would often fail and causes significant delays in ready states or even cause timeout errors.
+
+As its purpose is contradictory to Discord's intentions to reduce scraping of user and presence data, it has been removed.
+
+#### ClientOptions#messageEditHistoryMaxSize
+
+The `ClientOptions#messageEditHistoryMaxSize` option has been removed.
+
+To reduce caching, discord.js will no longer store an edit history. You will need to implement this yourself if required.
+
+### ClientUser
+
+#### ClientUser#setActivity
+
+This `ClientUser#setActivity` method no longer returns a Promise.
+
+#### ClientUser#setAFK
+
+This `ClientUser#setAFK` method no longer returns a Promise.
+
+#### ClientUser#setPresence
+
+This `ClientUser#setPresence` method no longer returns a Promise.
+
+`PresenceData#activity` is replaced with `PresenceData#activities` which now requires an `Array<ActivitiesOptions>`.
```

Comment:
Shouldn't this go below setActivity instead of setPresence?

Reply:
No, or at least not yet, because setActivity doesnt accept `PresenceData`... or an array of ActivityOptions.